### ISSUE TITLE
(Bug report) Markdown syntax highlighting not working

### ISSUE THREAD

**Issue by surajsharma** [TriliumNext/Trilium#3285]
### Trilium Version

0.56.2

### What operating system are you using?

macOS

### What is your setup?

Local + server sync

### Operating System Version

macOS Monterey 12.6 (21G115)

### Description

markdown notes are no longer syntax-highlighted as of 0.56.2

**Comment by zadam** [comment]
Hi, seems to work for me:

![image](https://user-images.githubusercontent.com/617641/199826255-ec45d199-4bd9-41e8-aeab-29dcd8f0841a.png)

**Comment by surajsharma** [comment]
> Hi, seems to work for me:
> 
> ![image](https://user-images.githubusercontent.com/617641/199826255-ec45d199-4bd9-41e8-aeab-29dcd8f0841a.png)

not working for me in any browser, what gives?

**Comment by sigaloid** [comment]
Try pressing F12 and reloading the page - see if there's any failed network requests.

Alternatively try changing the font, maybe the font for bold and italic variants aren't loading for some reason.

**Comment by surajsharma** [comment]
> Try pressing F12 and reloading the page - see if there's any failed network requests.
> 
> Alternatively try changing the font, maybe the font for bold and italic variants aren't loading for some reason.

did all that to no avail, the server instance runs fine though

**Comment by zadam** [comment]
@surajsharma are you by chance using master version and not released artifacts?

**Comment by surajsharma** [comment]
> @surajsharma are you by chance using master version and not released artifacts?

yes, oops? is there another branch i can use?

**Comment by zadam** [comment]
@surajsharma yes, `stable` branch. Master is used for development and can be unstable.
